hs_core/models.py

```diff
@@ -3349,7 +3357,7 @@ def size(self):
         Raises SessionException if iRODS fails.
         """
         # compute the total file size for the resource
-        f_sizes = [f.size for f in self.files.all()]
+        f_sizes = [f.size() for f in self.files.all()]
```

Comment:
We should do the sum at the DB level. Something like: 
`res_size = self.files.aggregate(Sum('_size'))`
`return res_size`

'Sum' needs to be imported from django.db.models

Reply:
done